cs/common/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required (VERSION 3.8)
+project (cs_common LANGUAGES CSharp)
+
+FILE(GLOB_RECURSE src *.cs)
```

Comment:
>* [](start = 22, length = 1)

All other cmake files seem to use an explicit file list.

Reply:
too many files (especially for the other projects)

---
In reply to: [192496917](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192496917) [](ancestors = 192496917)